Views/AsyncEditor.Editor.cshtml

```diff
@@ -44,3 +44,21 @@
         </div>
     }
 </div>
+
+<script>
+    jQuery(function ($) {
+        // Prevent multi submissions on forms.
+        $("body").on("submit", "form.no-multisubmit", function (e) {
+            const submittingClass = "submitting";
+            form = $(this);
+
+            if (form.hasClass(submittingClass)) {
+                e.preventDefault();
+                return;
+            }
+
+            form.addClass(submittingClass);
+            $.lombiq_LoadingIndicator.show();
+        });

```

Comment:
Ha validation error van a kliensoldalon, akkor tudok újra submitelni (lehet, hogy akkor a submit event nem is aktiválódik)? Szerveroldal esetében is ellenőrizni kell - nem emlékszem, hogy akkor a teljes editort tölti-e újra.

Reply:
> Ha szerveroldali hiba van, akkor submitnál a loading indicator miatt nem tudsz újra rányomni és utána az egész editor újratölt, tehát így akadályozzuk meg a multisubmitet?

Így igaz.

>  Leginkább az érdekel, hogy ez az összes editorra így működik-e, mert akkor oké.

Nem tudom hogy a jelenlegi összesnél meg van-e oldva. Amit néztem ott volt loading indicator. Ha azt szeretnénk hogy minden form-on legyen loading indicator, akkor valami olyasmit lehet csinálni gondolom hogy `$("form").on("submit", /*loading indicator hívása*/);` (tehát a form element-re a selector) és ez minden page-en legyen beinclude-olva. 
<sub>(Vagy valahogy megoldani hogy csak ahol van form.) Esetleg valami filter és ha a filter context Result valami Async Editor-os, akkor include-olni valahogy, hogy csak Async Editor-os esetekben legyen ez? </sub>
Nekem az a logikus, hogy ha szeretnénk loading indicatort, akkor hozzáadjuk, de elfogadom hogy ha ez nem jó.

Reply:
@aracsibalint A megoldásod valószínűleg jó lesz. Légyszi teszteld végig az összes editoron és ahol esetleg van override ott is alkalmazd. Ha ehhez segítség kell, nyugodtan pingelhetsz.